Automattic/wp-calypso
# Notifications: Improve saving UX

Closes 133-gh-Automattic/dotcom-feedback.

## Proposed Changes

Improves the UX when saving notifications. This adds a loading state to the buttons and disables checkbox actions whenever the notifications are being saved.

https://github.com/user-attachments/assets/6cb85959-23ae-4024-a7dd-f30dcb467e72

## Testing Instructions

Go to `/me/notifications` and tweak settings for sites. You should notice that the checkbox gets disabled, as well as the buttons. Additionally, there's a loading indicator.
